### PR TITLE
Instructions to enable Azure application insights

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: web-app/pom.xml [hunk 1]
```diff
@@ -21,8 +21,25 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.test.skip>true</maven.test.skip>
     </properties>
+    <repositories>
```
Comment (xscript) [inline web-app/pom.xml]: This should be unnecessary as it is one of the default repo in Maven.

Comment (xiangyan99) [reply web-app/pom.xml]: This is documented in our official document https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started. I prefer to keep them to make sure our demo align with our doc to not confuse our customers. And I am open to remove it too. Let me know your suggestions.

Comment (selvasingh) [reply web-app/pom.xml]: Thanks for the contribution. Like to know if this is an optional or required step?

Comment (xiangyan99) [reply web-app/pom.xml]: It is optional. If you don't specify a valid instrumentation key, AI will be disabled by default.

**Pull request merged**